src/ng-s3upload/directives/s3-upload.js

```diff
@@ -11,10 +11,8 @@ angular.module('ngS3upload.directives', []).
         $scope.success = false;
         $scope.uploading = false;
 
-        $scope.barClass = function () {
-          return {
-            "bar-success": $scope.attempt && !$scope.uploading && $scope.success
-          };
+        $scope.isUploadSuccessful = function(){
```

Comment:
The issue with the previous implementation was the following:
- Bootstrap2 `active` and `success` class are not on the same element
- on the bootstrap3 theme, there was a duplicated `ng-class` attribute which caused one of them to be ignored.
As the included templates are strongly tied to boostrap, and since we can now provide a custom template, I think it makes more sense to provide utility values, such as `uploaded` `uploadSuccess` to ease custom template creation.